db-engine/src/main/java/com/splicemachine/db/impl/jdbc/EmbedConnectionMaker2.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2012 - 2021 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.splicemachine.tools;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Properties;
+
+import com.splicemachine.db.jdbc.EmbeddedDriver;
+
+
+/**
+ * Provides connections for internal use.
+ */
+public final class EmbedConnectionMaker2 {
```

Comment:
EmbedConnectionMaker2 is a version that db-engine has access because this project can't have a dependency on splice_machine.  I couldn't remove EmbedConnectionMaker because db-engine can't see SConfiguration.

Reply:
I created https://splicemachine.atlassian.net/browse/DB-11493 , I think we should make an effort to clean it up at some point.